src/seedu/addressbook/AddressBook.java

```diff
@@ -1144,14 +1152,14 @@ private static String getUsageInfoForExitCommand() {
      */
 
     /**
-     * Removes sign(p/, d/, etc) from parameter string
+     * Removes all the occurrences of the substring from parameter fullString
      *
-     * @param s  Parameter as a string
-     * @param sign  Parameter sign to be removed
-     * @return  string without the sign
+     * @param fullString  Parameter as a string
+     * @param subString to be removed
+     * @return  fullString without the substring
      */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
+    private static String removeAllSubstrings(String fullString, String subString) {
```

Comment:
Yes, you change the method name to reflect what the method does. But what is needed here in addressbook is a method to remove prefix in a string right? :P